br/pkg/lightning/backend/local/duplicate.go

```diff
@@ -809,7 +809,7 @@ func (m *DupeDetector) buildLocalDupTasks(dupDB *pebble.DB, keyAdapter common.Ke
 }
 
 // CollectDuplicateRowsFromDupDB collects duplicates from the duplicate DB and records all duplicate row info into errorMgr.
-func (m *DupeDetector) CollectDuplicateRowsFromDupDB(ctx context.Context, dupDB *pebble.DB, keyAdapter common.KeyAdapter, algorithm config.DuplicateResolutionAlgorithm) error {
+func (m *DupeDetector) CollectDuplicateRowsFromDupDB(ctx context.Context, dupDB *pebble.DB, keyAdapter common.KeyAdapter, algorithm string) error {
```

Comment:
can we use an enum type to represent the algorithm of duplicate record handling mode？

Reply:
Originally `conflict.strategy` was in the type of `string`, so I maintained it in the old code. However, using enum type indeed is clearer, so I have changed the type to enum.